src/app/games/game-details/game-details.component.html

```diff
@@ -35,34 +35,71 @@
             </div>
         </div>
     </div>
-    <div class='row'>
-        <div class="col-md-2 col-sm-12">
+    <div class="data-row">
+        <div>
             <strong>When?</strong>
         </div>
-        <div *ngIf="!editMode" class="col-md-10 col-sm-12">
+        <div *ngIf="!editMode">
             {{ game?.datetime | date: 'shortTime' }}
             {{ game?.datetime | date: 'fullDate' }}
         </div>
-        <div *ngIf="editMode" class="col-md-10 col-sm-12">
+        <div *ngIf="editMode">
```

Comment:
This is just some legacy trash code. It's safe to remove. I think we should make game edit screen the same way we have it for teams now.